### PR TITLE
Add security researcher opening

### PR DESCRIPTION
**Pull request by blancoagostina**

### CODE REVIEW

File: _jobs/Security-Researcher.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: 'Security Researcher'
+location: 'Global'
+type: 'Full Time'
+team: 'Engineering'
+navigation_weight: 0
+tags:
+  - engineering
+date: 2019-11-05 19:07:50 +0000
+sub_heading: ' '
+description: 'We proudly build the most widely used open source infrastructure tools: the OpenZeppelin smart contracts library powers 3000 public projects and has over 6500 Github stars, while the ZeppelinOS development toolset is leading the way in upgradeability and secure code reuse.'
```
Comment (elopio) [inline _jobs/Security-Researcher.md]:
```suggestion
description: 'We proudly build the most widely used open source infrastructure tools: the OpenZeppelin smart contracts library powers 3000 public projects and has over 6500 Github stars, while the OpenZeppelin development toolset is leading the way in upgradeability and secure code reuse.'
```

File: _jobs/Security-Researcher.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <li><p>Write detailed, engaging reports on the vulnerabilities of those systems.</p></li>
+    <li><p>Conduct open-ended research around cutting edge blockchain technologies.</p></li>
+  </ul>
+</div>
+<div class="requirements">
+  <h3 class="job-description-title">Who you are:</h3>
+  <ul>
+    <li><p>You are extremely detail oriented.</p></li>
+    <li><p>You are a self-starter that works independently, with minimal supervision.</p></li>
+    <li><p>You possess an advanced english level and great communication skills (oral and written).</p></li>
```
Comment (elopio) [inline _jobs/Security-Researcher.md]:
```suggestion
    <li><p>You possess an advanced English level and great communication skills (oral and written).</p></li>
```

**Comment by elopio** [comment]
It's weird because on the preview I see a different text:
https://deploy-preview-74--openzeppelin.netlify.com/jobs/security-researcher/
@blancoagostina any idea why?

**Comment by blancoagostina** [comment]
> It's weird because on the preview I see a different text:
> https://deploy-preview-74--openzeppelin.netlify.com/jobs/security-researcher/
> @blancoagostina any idea why?

Actually that is not being used, it was part from the old design. I'll delete the descriptions from all of the openings

**Pull request merged**